### PR TITLE
Add support for DeathAdder V3 Pro (00c2/00c3)

### LINKED ISSUE #2214: Support for DeathAdder V3 Pro (2024)
### Device Info

* Name: DeathAdder V3 PRO
* Product Number: RZ01-04630300
* Model Number: RZ01-0463

RazerGenie cannot detect the device. I noticed the PID is different from the currently supported version of this device.

### Device Information
```
Wireless:
Bus 003 Device 002: ID 1532:00c3 Razer USA, Ltd Razer DeathAdder V3 Pro
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 [unknown]
  bDeviceSubClass         0 [unknown]
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x00c3 Razer DeathAdder V3 Pro
  bcdDevice            1.01
  iManufacturer           1 Razer
  iProduct                2 Razer DeathAdder V3 Pro
  iSerial                 0 
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0054
    bNumInterfaces          3
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0xa0
      (Bus Powered)
      Remote Wakeup
    MaxPower              100mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      2 Mouse
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      0 
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     260
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0010  1x 16 bytes
        bInterval               4
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      61
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               4

Wired:
Bus 001 Device 007: ID 1532:00c2 Razer USA, Ltd Razer DeathAdder V3 Pro
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 [unknown]
  bDeviceSubClass         0 [unknown]
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x00c2 Razer DeathAdder V3 Pro
  bcdDevice            1.00
  iManufacturer           1 Razer
  iProduct                2 Razer DeathAdder V3 Pro
  iSerial                 3 000000000000
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0054
    bNumInterfaces          3
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0xe0
      Self Powered
      Remote Wakeup
    MaxPower              500mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      2 Mouse
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     159
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      61
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               1

Bus 003 Device 005: ID 1532:00c3 Razer USA, Ltd Razer DeathAdder V3 Pro
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 [unknown]
  bDeviceSubClass         0 [unknown]
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x00c3 Razer DeathAdder V3 Pro
  bcdDevice            1.01
  iManufacturer           1 Razer
  iProduct                2 Razer DeathAdder V3 Pro
  iSerial                 0 
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0054
    bNumInterfaces          3
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0xa0
      (Bus Powered)
      Remote Wakeup
    MaxPower              100mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      2 Mouse
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      0 
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     260
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0010  1x 16 bytes
        bInterval               4
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.00
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      61
          Report Descriptors: 
            ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               4


```

```
Wireless:
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-event-if01 -> ../event6
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-event-mouse -> ../event3
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-if01-event-kbd -> ../event4
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-if01-event-mouse -> ../event5
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-if01-mouse -> ../mouse1
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-if02-event-kbd -> ../event7
lrwxrwxrwx 1 root root   9 Mar  6 14:50 usb-Razer_Razer_DeathAdder_V3_Pro-mouse -> ../mouse0

Wired:
lrwxrwxrwx 1 root root  10 Mar  6 15:32 usb-Razer_Razer_DeathAdder_V3_Pro_000000000000-event-if01 -> ../event26
lrwxrwxrwx 1 root root  10 Mar  6 15:32 usb-Razer_Razer_DeathAdder_V3_Pro_000000000000-event-mouse -> ../event24
lrwxrwxrwx 1 root root  10 Mar  6 15:32 usb-Razer_Razer_DeathAdder_V3_Pro_000000000000-if01-event-kbd -> ../event25
lrwxrwxrwx 1 root root  10 Mar  6 15:32 usb-Razer_Razer_DeathAdder_V3_Pro_000000000000-if02-event-kbd -> ../event27
lrwxrwxrwx 1 root root   9 Mar  6 15:32 usb-Razer_Razer_DeathAdder_V3_Pro_000000000000-mouse -> ../mouse3
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-event-if01 -> ../event6
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-event-mouse -> ../event7
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-if01-event-kbd -> ../event4
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-if01-event-mouse -> ../event5
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-if01-mouse -> ../mouse0
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-if02-event-kbd -> ../event3
lrwxrwxrwx 1 root root   9 Mar  6 14:59 usb-Razer_Razer_DeathAdder_V3_Pro-mouse -> ../mouse1


```

<details>

```
Wireless:
003:005:002:DESCRIPTOR         1709755164.999908
 05 01 09 06 A1 01 05 07 19 E0 29 E7 15 00 25 01
 75 01 95 08 81 02 81 01 19 00 2A FF 00 15 00 26
 FF 00 75 08 95 06 81 00 05 08 19 01 29 03 15 00
 25 01 75 01 95 03 91 02 95 05 91 01 C0

003:005:001:DESCRIPTOR         1709755165.000280
 05 01 09 06 A1 01 85 01 05 07 19 E0 29 E7 15 00
 25 01 75 01 95 08 81 02 19 00 2A FF 00 15 00 26
 FF 00 75 08 95 0E 81 00 05 08 19 01 29 03 15 00
 25 01 75 01 95 03 91 02 95 05 91 01 C0 05 01 09
 02 A1 01 85 0A 09 01 A1 00 05 09 19 01 29 06 15
 00 25 01 75 01 95 06 81 02 75 01 95 02 81 03 06
 00 FF 09 40 75 08 95 02 15 81 25 7F 81 02 05 01
 09 38 15 81 25 7F 75 08 95 01 81 06 09 30 09 31
 16 00 80 26 FF 7F 75 10 95 02 81 06 C0 C0 05 0C
 09 01 A1 01 85 02 19 00 2A 3C 02 15 00 26 3C 02
 95 01 75 10 81 00 75 08 95 0D 81 01 C0 05 01 09
 80 A1 01 85 03 19 81 29 83 15 00 25 01 75 01 95
 03 81 02 95 05 81 01 75 08 95 0E 81 01 C0 05 01
 09 00 A1 01 85 04 09 03 15 00 26 FF 00 35 00 46
 FF 00 75 08 95 0F 81 00 C0 05 01 09 00 A1 01 85
 05 09 03 15 00 26 FF 00 35 00 46 FF 00 75 08 95
 0F 81 00 C0

003:005:000:DESCRIPTOR         1709755165.000905
 05 01 09 02 A1 01 09 01 A1 00 05 09 19 01 29 05
 15 00 25 01 75 01 95 05 81 02 75 01 95 0B 81 03
 05 0C 0A 38 02 15 81 25 7F 75 08 95 01 81 06 05
 01 09 38 15 81 25 7F 75 08 95 01 81 06 09 30 09
 31 16 01 80 26 FF 7F 75 10 95 02 81 06 C0 06 00
 FF 09 02 15 00 25 01 75 08 95 5A B1 01 C0

Wired:
003:005:002:DESCRIPTOR         1709757682.873006
 05 01 09 06 A1 01 05 07 19 E0 29 E7 15 00 25 01
 75 01 95 08 81 02 81 01 19 00 2A FF 00 15 00 26
 FF 00 75 08 95 06 81 00 05 08 19 01 29 03 15 00
 25 01 75 01 95 03 91 02 95 05 91 01 C0

003:005:001:DESCRIPTOR         1709757682.873368
 05 01 09 06 A1 01 85 01 05 07 19 E0 29 E7 15 00
 25 01 75 01 95 08 81 02 19 00 2A FF 00 15 00 26
 FF 00 75 08 95 0E 81 00 05 08 19 01 29 03 15 00
 25 01 75 01 95 03 91 02 95 05 91 01 C0 05 01 09
 02 A1 01 85 0A 09 01 A1 00 05 09 19 01 29 06 15
 00 25 01 75 01 95 06 81 02 75 01 95 02 81 03 06
 00 FF 09 40 75 08 95 02 15 81 25 7F 81 02 05 01
 09 38 15 81 25 7F 75 08 95 01 81 06 09 30 09 31
 16 00 80 26 FF 7F 75 10 95 02 81 06 C0 C0 05 0C
 09 01 A1 01 85 02 19 00 2A 3C 02 15 00 26 3C 02
 95 01 75 10 81 00 75 08 95 0D 81 01 C0 05 01 09
 80 A1 01 85 03 19 81 29 83 15 00 25 01 75 01 95
 03 81 02 95 05 81 01 75 08 95 0E 81 01 C0 05 01
 09 00 A1 01 85 04 09 03 15 00 26 FF 00 35 00 46
 FF 00 75 08 95 0F 81 00 C0 05 01 09 00 A1 01 85
 05 09 03 15 00 26 FF 00 35 00 46 FF 00 75 08 95
 0F 81 00 C0

003:005:000:DESCRIPTOR         1709757682.874125
 05 01 09 02 A1 01 09 01 A1 00 05 09 19 01 29 05
 15 00 25 01 75 01 95 05 81 02 75 01 95 0B 81 03
 05 0C 0A 38 02 15 81 25 7F 75 08 95 01 81 06 05
 01 09 38 15 81 25 7F 75 08 95 01 81 06 09 30 09
 31 16 01 80 26 FF 7F 75 10 95 02 81 06 C0 06 00
 FF 09 02 15 00 25 01 75 08 95 5A B1 01 C0

001:007:002:DESCRIPTOR         1709757682.875752
 05 01 09 06 A1 01 05 07 19 E0 29 E7 15 00 25 01
 75 01 95 08 81 02 81 01 19 00 2A FF 00 15 00 26
 FF 00 75 08 95 06 81 00 05 08 19 01 29 03 15 00
 25 01 75 01 95 03 91 02 95 05 91 01 C0

001:007:001:DESCRIPTOR         1709757682.880749
 05 01 09 06 A1 01 85 01 05 07 19 E0 29 E7 15 00
 25 01 75 01 95 08 81 02 19 00 2A FF 00 15 00 26
 FF 00 75 08 95 0E 81 00 C0 05 0C 09 01 A1 01 85
 02 19 00 2A 3C 02 15 00 26 3C 02 95 01 75 10 81
 00 75 08 95 0D 81 01 C0 05 01 09 80 A1 01 85 03
 19 81 29 83 15 00 25 01 75 01 95 03 81 02 95 05
 81 01 75 08 95 0E 81 01 C0 05 01 09 00 A1 01 85
 04 09 03 15 00 26 FF 00 35 00 46 FF 00 75 08 95
 0F 81 00 C0 05 01 09 00 A1 01 85 05 09 03 15 00
 26 FF 00 35 00 46 FF 00 75 08 95 0F 81 00 C0

001:007:000:DESCRIPTOR         1709757682.884747
 05 01 09 02 A1 01 09 01 A1 00 05 09 19 01 29 05
 15 00 25 01 75 01 95 05 81 02 75 01 95 03 81 03
 06 00 FF 09 40 75 08 95 02 15 81 25 7F 81 02 05
 01 09 38 15 81 25 7F 75 08 95 01 81 06 09 30 09
 31 16 00 80 26 FF 7F 75 10 95 02 81 06 C0 06 00
 FF 09 02 15 00 25 01 75 08 95 5A B1 01 C0


```

</details>

### Packet Captures

<Please see here as we probably need packet captures>
<https://github.com/openrazer/openrazer/wiki/Reverse-Engineering-USB-Protocol>

### Synapse Screenshots
<Screenshots of all available options as seen on **Razer Synapse** in a Windows Environment>

### PR DESCRIPTION
**Pull request by ryan-1337**
Fixes #2214

### CODE REVIEW

**Comment by z3ntu** [comment]
Does any marketing material / names actually include "2024" in the name? If not please remove that from the name here, we already have several of such instances with other devices.

Also could you share here again what the status of this is? You wrote before "it's not detected by polychromatic" but do for example the udev rules trigger, the kernel driver bind, etc?

**Comment by ryan-1337** [comment]
Can we rename the device to Deathadder v3 pro Smooth as described on the razer website instead of deathadder v3 pro 2024? 

On my machine I can't load the drivers but on a vm everything loads fine.

**Comment by z3ntu** [comment]
I don't see the device being named that way? Sure, you can select "Smooth-Touch" as finish but it's not a part of the device name, and the "Classic Base" surely would be the same software-wise.

image

So I think just "DeathAdder V3 Pro" like the device that's already supported is fine. I think other devices with the same "problem" use e.g. `_ALT` in the driver (Ornata V3 X)

**Comment by z3ntu** [comment]
Why did you rename the device after I wrote https://github.com/openrazer/openrazer/pull/2239#issuecomment-2081628660 ?

**Comment by ryan-1337** [comment]
> Why did you rename the device after I wrote [#2239 (comment)](https://github.com/openrazer/openrazer/pull/2239#issuecomment-2081628660) ?

Sorry, I didn't quite understand.
it's fixed

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 1]
```diff
@@ -1566,6 +1566,37 @@ class RazerDeathAdderV3ProWireless(RazerDeathAdderV3ProWired):
...
+    USB_PID = 0x00C2
+    METHODS = ['get_device_type_mouse', 'max_dpi', 'get_dpi_xy', 'set_dpi_xy', 'get_dpi_stages', 'set_dpi_stages',
+               'get_poll_rate', 'set_poll_rate', 'get_supported_poll_rates',
+               'get_battery', 'is_charging', 'get_idle_time', 'set_idle_time', 'set_low_battery_threshold']
+
+    DEVICE_IMAGE = "https://dl.razerzone.com/src/6130/6130-1-en-v2.png"
+
+    DPI_MAX = 30000
+
+    POLL_RATES = [125, 500, 1000]
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py]: It doesn't support 8000 Hz polling rate when wired? Only wireless?

**Comment by z3ntu** [comment]
So functionality-wise everything works correctly? Any functionality missing?

File: daemon/openrazer_daemon/hardware/mouse.py (hunk 1 above)
Comment (ryan-1337) [reply daemon/openrazer_daemon/hardware/mouse.py]: yes 8k wireless only

**Comment by ryan-1337** [comment]
> So functionality-wise everything works correctly? Any functionality missing?

yes, everything looks good to me

File: daemon/openrazer_daemon/hardware/mouse.py (hunk 1 above)
Comment (z3ntu) [reply daemon/openrazer_daemon/hardware/mouse.py]: weird but okay

**Comment by z3ntu** [comment]
I've now tried to align the code more with the other DeathAdder V3 Pro since those two should have identical functionality.

By doing this for now I've removed the 8kHz polling rate and hyperpolling led indicator, let's add this later.

I'll merge this for now assuming it still works but please try the latest state also and let me know if it does or doesn't work.

**Comment by z3ntu** [comment]
For the other functionality please pull [`deathadder_v3_pro_functionality`](https://github.com/openrazer/openrazer/commits/deathadder_v3_pro_functionality/) into a branch of yours and check if it works still also, then open a PR from that. Thanks!

**Comment by ryan-1337** [comment]
> I've now tried to align the code more with the other DeathAdder V3 Pro since those two should have identical functionality.
> 
> By doing this for now I've removed the 8kHz polling rate and hyperpolling led indicator, let's add this later.
> 
> I'll merge this for now assuming it still works but please try the latest state also and let me know if it does or doesn't work.

Hello, after trying it, the mouse is no longer detected
Screenshot from 2024-07-03 23-23-42

**Comment by z3ntu** [comment]
Bah, nothing important was changed that should be able to cause this.

Are you sure you built correctly from master branch? Please check `grep -i 00c3 /lib/udev/rules.d/99-razer.rules` and dmesg for anything related to razermouse

**Comment by ryan-1337** [comment]
ok it works mb problem of permissions i had to add this line 
``
sudo chown -R root:plugdev /sys/devices
``

**Pull request merged**